src/main/java/teammates/common/util/Const.java

```diff
@@ -811,6 +811,8 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
         public static final String COMMENT_ID = "commentid";
         public static final String COMMENT_TEXT = "commenttext";
         
+        public static final String SEND_SUBMISSION_EMAIL = "isSendSubmissionEmail";
```

Comment:
use all lowercase and drop the `is-` prefix for this, i.e. `sendsubmissionemail`.